comp/core/tagger/remote/tagger.go

```diff
@@ -400,7 +410,3 @@ func convertEventType(t pb.EventType) (types.EventType, error) {
 func convertEntityID(id *pb.EntityId) string {
 	return fmt.Sprintf("%s://%s", id.Prefix, id.Uid)
 }
-
```

Comment:
Do you know what this code does?

Reply:
It seems that this code is to initialize the grpclog which should be removed in this pr. We need to check if it is already initialized in workloadmeta collectors [init()](https://github.com/DataDog/datadog-agent/blob/d5b2c0a393fa77b4e6115d38bff6e5fa80c5b8a2/comp/core/workloadmeta/collectors/internal/remote/workloadmeta/workloadmeta.go#L77). But we should probably remove that init() as well.